cg/store/api/find_business_data.py

```diff
@@ -63,29 +63,29 @@ def analyses(self, *, family: Family = None, before: dt.datetime = None) -> Quer
             ).filter(Analysis.started_at < before)
         return records
 
-    def active_sample(self, internal_id: str) -> bool:
+    def is_active_sample(self, internal_id: str) -> bool:
         """Check if there are any active cases for a sample"""
         sample: Sample = self.get_sample_by_internal_id(internal_id=internal_id)
         if any(
             [
-                self.family(
-                    internal_id=self.Family.query.filter(Family.id == family_sample.family_id)
-                    .first()
-                    .internal_id
-                ).action
-                == "analyze"
-                or self.family(
-                    internal_id=self.Family.query.filter(Family.id == family_sample.family_id)
-                    .first()
-                    .internal_id
-                ).action
-                == "running"
+                self.is_family_action(family_id=family_sample.family_id, action="analyze")
+                or self.is_family_action(family_id=family_sample.family_id, action="running")
```

Comment:
We have a sample object here, so I think we can just trace back the families with something like this would be clearer than going via the ID.
```python
    @property
    def cases(self) -> List[Family]:
        """Return the cases connected to the sample."""
        return self._get_cases

    @property
    def _get_cases(self) -> List[Family]:
        """Extract cases from a sample."""
        return [link.family for link in self.links]
```

Reply:
This can be done in the family refactor @seallard I just tried to make the code clearer for now